tests/test_other.py

```diff
@@ -4880,9 +4880,9 @@ def test_bad_locale(self):
 ''')
     self.run_process([EMXX, 'src.cpp'])
 
-    self.assertContained('locale set to C: C;C;C;C;C;C',
+    self.assertContained('locale set to C: C',
                          self.run_js('a.out.js', args=['C']))
-    self.assertContained('locale set to waka: waka;waka;waka;waka;waka;waka',
+    self.assertContained('locale set to waka: waka',
```

Comment:
why did the output change here?

Reply:
I can't remember exactly but I think this was either bug in the existing version of musl or some detail that is allowed to vary.

Certainly if you run this under glibd you don't get the repeated field:

```
$ clang test.c
$ ./a.out 
locale set to C: C
```

So I think this is a bugfix, or at least harmless.